erlang/p2p/node.erl

```diff
@@ -0,0 +1,52 @@
+-module(node).
+
+-export([test/1]).
+
+-include("../blocktree/blocktree.hrl").
+-include_lib("stdlib/include/assert.hrl").
+
+test(Tp) ->
+    io:fwrite("hello, world\n"),
+    if Tp == 1 ->
```

Comment:
can use client and server instead of 1 and 2

Reply:
👍